app/views/layouts/application.html.erb

```diff
@@ -17,6 +17,8 @@
         <li><%= link_to "Home", root_path %></li>
         <li><%= link_to "Recipes", recipes_path %></li>
         <li><%= link_to "New Recipe", new_recipe_path %></li>
+        <li><%= link_to "Foods", foods_path %></li>
+        <li><%= link_to "New Food", new_food_path %></li>
```

Comment:
Nice job adding those elements to the menu bar. Consider adding as well a link to the public recipes and the shopping list.

Reply:
Thank you for your suggestions. Surely I will add it but still, there is some work left to implement completely Public recipes and shopping list that's why I avoided adding them at this stage